metrics/metrics/src/test/java/io/helidon/metrics/TestNearestValueSearch.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.helidon.metrics;
+
+import io.helidon.metrics.WeightedSnapshot.WeightedSample;
+import org.junit.jupiter.api.Test;
+
+import static io.helidon.metrics.Sample.derived;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class TestNearestValueSearch {
```

Comment:
Test classes can be package private.
Test methods can be package private.

Reply:
Fixed